frontend/src/components/Metrics/MetricsStep/Classification/index.tsx

```diff
@@ -16,31 +16,26 @@ interface classificationProps {
 
 export const Classification = ({ options, title, label }: classificationProps) => {
   const dispatch = useAppDispatch()
-  const importClassification = useAppSelector(selectMetricsContent).classification
   const isProjectCreated = useAppSelector(selectMetricsContent).isProjectCreated
-  const optionsName = options.map((e) => e.name)
-
-  const defaultInput = getArrayIntersection(optionsName, importClassification)
-  const [selectedTargetField, setSelectedTargetField] = useState(isProjectCreated ? [] : defaultInput)
-  const isAllSelected = selectedTargetField.length > 0 && selectedTargetField.length === options.length
+  const importClassification = useAppSelector(selectMetricsContent).classification
+  const targetFieldNames = options.map((e) => e.name)
```

Comment:
should select key of targetFields to compare with classification, because we saved targetfIelds.key when click save button